dist/init/systemd/rkt-gc.service

```diff
@@ -1,3 +1,7 @@
+[Unit]
+Description=Garbage Collection for rkt
+
 [Service]
+Environment=GRACE_PERIOD=1d
```

Comment:
We now do networking cleanup only in GC (it used to also be done on pod exit). Having IPs linger around for 1 day can easily lead to IP exhaustion. Not to mention the memory overhead of the stacks. Can we drop this at least to 1 hour? If there's nothing for GC to do, running it is pretty cheap.